packages/docusaurus-utils/package.json

```diff
@@ -19,6 +19,7 @@
   "license": "MIT",
   "dependencies": {
     "@docusaurus/logger": "3.0.0",
+    "@docusaurus/utils-common": "3.0.0",
```

Comment:
Do we need that new dependency here?
It looks like it's only for the unit test that shouldn't be there